packages/cli/cli_internal.ts

```diff
@@ -209,6 +216,7 @@ nftCommand
   .option('-d, --localDir <directory-of-local-images>', 'directory of local images to be uploaded')
   .option('-i, --ipfsDir <ipfs-directory-of-uploaded-images>', 'IPFS directory to upload the images')
   .option('-m, --metadataFile <metadata-file>', 'file to store the metadata of the uploaded images')
+  .option('--debug', 'show detailed debug information such as error stack traces')
```

Comment:
Let's call `enableDebugMode` as well when this is enabled, so that all of the API queries would be logged as well.

Not sure the log would look nice, otherwise please feel free to improve it.

Reply:
Done